### PR TITLE
Add Cloud SQL MySQL connectivity samples for SQLAlchemy. 

### PR DESCRIPTION
**Pull request by kurtisvg**
New sample app in a series of samples designed to highlight connection best practices. The app is a simple web app that allows voting between two different groups ("Tabs" vs "Spaces") and records the votes in a MySQL backend. 

Java equivalent is [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/1231).

### CODE REVIEW

File: cloud-sql/mysql/sqlalchemy/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+1. Use the information noted in the previous steps:
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service/account/key.json
+export CLOUD_SQL_CONNECTION_NAME='<MY-PROJECT>:<INSTANCE-REGION>:<MY-DATABASE>'
+export DB_USER='my-db-user'
+export DB_PASS='my-db-pass'
+export DB_NAME='my_db'
+```
+Note: Saving credentials in environment variables is convenient, but not secure - consider a more
```
Comment (tswast) [inline cloud-sql/mysql/sqlalchemy/README.md]: 12 factor app might disagree with you for this. https://12factor.net/config

It all depends on how you set the environment variables. Even Kubernetes secrets can be used to set environment variables, and I wouldn't call that "not secure".

Plus, KMS can't help you store your secrets, only encrypt/decrypt them.

File: cloud-sql/mysql/sqlalchemy/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Finally, start the application:
+```bash
+python main.py
+```
+
+Navigate towards `http://127.0.0.1:8080` to verify your application is running correctly.
+
+## Google App Engine Standard
+
+To run on GAE-Standard, create an AppEngine project by following the setup for these
```
Comment (tswast) [inline cloud-sql/mysql/sqlalchemy/README.md]: space between App Engine

File: cloud-sql/mysql/sqlalchemy/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+export DB_USER='my-db-user'
+export DB_PASS='my-db-pass'
+export DB_NAME='my_db'
+```
+Note: Saving credentials in environment variables is convenient, but not secure - consider a more
+secure solution such as [Cloud KMS](https://cloud.google.com/kms/) to help keep secrets safe.
+
+## Running locally
+
+To run this application locally, download and install the cloud_sql_proxy by
```
Comment (tswast) [inline cloud-sql/mysql/sqlalchemy/README.md]: backticks around `cloud_sql_proxy` or use the proper name (Cloud SQL proxy)

File: cloud-sql/mysql/sqlalchemy/main.py [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+import datetime
+import logging
+import os
+
+from flask import Flask, render_template, request, Response
+import sqlalchemy
+
+
+# Saving credentials in environment variables is convent, but not secure
```
Comment (tswast) [inline cloud-sql/mysql/sqlalchemy/main.py]: Ditto. This isn't the worst practice, and might actually offend the 12-factor app developer community. This comment _is_ appropriate in the `app.yaml` file, though. You don't want people checking secrets into version control.

File: cloud-sql/mysql/sqlalchemy/app.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: python37
+
+env_variables:
+  CLOUD_SQL_INSTANCE_NAME: <MY-PROJECT>:<INSTANCE-REGION>:<MY-DATABASE>
+  DB_USER: my-db-user
+  DB_PASS: my-db-pass
```
Comment (tswast) [inline cloud-sql/mysql/sqlalchemy/app.yaml]: Add a comment that storing passwords in version control is not secure. Suggesting you encrypt the password with KMS would be appropriate here.

File: cloud-sql/mysql/sqlalchemy/README.md (hunk 1 above)
Comment (kurtisvg) [reply cloud-sql/mysql/sqlalchemy/README.md]: I have to disagree about the security of store secrets in an environment variable. Environment variables are accesible by any process that runs in the user's environment, leaving them vulnerable. Any secret of significant value should not be stored as an environment variable. 

While KMS doesn't directly store secrets, they do have a page on how to store secrets using KMS: https://cloud.google.com/kms/docs/store-secrets

**Pull request merged**